src/signals/incident/definitions/wizard-step-2-vulaan/wegen-verkeer-straatmeubilair.js

```diff
@@ -190,6 +190,48 @@ export default {
       },
       render: FormComponents.MapSelect
     },
+    extra_straatverlichting_niet_op_kaart: {
+      meta: {
+        ifAllOf: {
+          subcategory: 'lantaarnpaal-straatverlichting',
+        },
+        ifOneOf: {
+          extra_straatverlichting_probleem: [
+            'lamp_doet_het_niet',
+            'lamp_brandt_overdag',
+            'geeft_lichthinder',
+            'lamp_is_vervuild',
+            'lamp_is_zichtbaar_beschadigd',
+            'overig'
+          ]
+        },
+        pathMerge: 'extra_properties',
+        value: 'Het lichtpunt staat niet op de kaart?'
+      },
+      render: FormComponents.CheckboxInput
+    },
+    extra_straatverlichting_niet_op_kaart_nummer: {
+      meta: {
+        label: 'Weet u het nummer dat op het lichtpunt staat?',
+        pathMerge: 'extra_properties',
+        placeholder: 'Nummer lichtpunt, nummer lichtpunt',
```

Comment:
`placeholder: 'Nummer lichtpunt` is beter denk ik. Als mensen daar een hele lijst invullen dan mag dat. Achterliggend is het een text veld zonder aanname hoe die in elkaar zit

Reply:
gedaan